pages/home/auth-providers/atlassian.mdx

```diff
@@ -5,7 +5,7 @@ import {Tabs} from 'nextra/components'
 <Note>
   At this time, Arcade does not offer a default Atlassian Auth Provider. To use
   Atlassian auth, you must create a custom Auth Provider with your own Atlassian
-  OAuth 2.0 credentials.
```

Comment:
Added "as described below" wording to every auth provider with this note. I didn't want the reader to think that this callout was describing something _other_ than what this page will teach them.